src/tests/ftest/pool/create_all_hw.yaml

```diff
@@ -0,0 +1,26 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+    - server-D
+    - server-E
+    - server-F
+    - server-G
```

Comment:
Do we really need HW Large - 8 server nodes? Could we use HW Medium with 4 dual-engine nodes and achieve the same test coverage?

Reply:
One purpose of the functional tests is to check the repartition of the pool storage and its metadata among the different engine. From what I understand of the metadata repartition, at  least 5 sever nodes should be used to properly tests the metadata duplication.

Reply:
At least 5 server nodes, or at least 5 engines? With HW Medium, we could use 4 dual-engine server nodes for a total of 8 engines.

Reply:
5 engines should be enough => thus HW medium should be enough.
- [ ] Use HW medium for funcitonal test

Reply:
Finally, I should have to stick to the large cluster => to properly test the available storage computation, I should have two NVMe devices per engine. My apologies, for the short lived happiness.